blazor/linear-gauge/getting-started.md

```diff
@@ -19,42 +19,41 @@ This section briefly explains about how to include [Blazor LinearGauge](https://
 
 ## Create a new Blazor App in Visual Studio
 
-You can create **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio in one of the following ways,
+You can create a **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio via [Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0) or the [Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-integration/template-studio).
 
-* [Create a Project using Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?pivots=windows&view=aspnetcore-7.0)
+## Install Syncfusion Blazor LinearGauge and Theme NuGet in the App
```

Comment:
Was theme package necessary?

Reply:
Theme package unnecessary so this package removed from this PR following by your guidance.